lib/src/widget/built_in/simple/simple.dart

```diff
@@ -44,28 +46,42 @@ class SimpleToastWidget extends StatelessWidget {
 
   final bool applyBlurEffect;
 
+  final bool? showCloseButton;
+  final VoidCallback? onCloseTap;
+
   SimpleStyle get defaultStyle => SimpleStyle(type);
 
   @override
-  Widget build(BuildContext context) {
-    final background = backgroundColor ?? defaultStyle.backgroundColor(context);
+  _SimpleToastWidgetState createState() => _SimpleToastWidgetState();
+}
 
+class _SimpleToastWidgetState extends State<SimpleToastWidget>
+    with SingleTickerProviderStateMixin {
```

Comment:
We don't need SingleTickerProviderStateMixin we only use this class when we want to use AnimationController so please remove it